src/exportstep.cpp

```diff
@@ -5,6 +5,235 @@
 //-----------------------------------------------------------------------------
 #include "solvespace.h"
 
+const double PRECISION = 0.001;
+
+//-----------------------------------------------------------------------------
+// Structs to keep track of duplicated entities
+//-----------------------------------------------------------------------------
+// Basic alias.
+typedef struct {
+    int reference;
+    List<int> aliases = {};
+} alias_t;
+
+// Cartesian points.
+typedef struct {
+    alias_t alias;
+    alias_t vertexAlias;
+    Vector v;
+} pointAliases_t;
+
+// Curves.
+typedef struct {
+    alias_t alias;
+    List<int> memberPoints = {};
+} curveAliases_t;
+
+// Edges.
+typedef struct {
+    alias_t alias;
+    int prevFinish;
+    int thisFinish;
+    int curveId;
+} edgeAliases_t;
+
+List<pointAliases_t> pointAliases;
+List<edgeAliases_t> edgeAliases;
+List<curveAliases_t> curveAliases;
+
+//-----------------------------------------------------------------------------
+// Functions for STEP export
+//-----------------------------------------------------------------------------
+// Check if this point was already defined with a different ID number.
+// inputs:
+//        number -> id of the cartesian point
+//        v -> position of the cartesian point
+//        vertex -> id of the vertex linked to this point (<1 if none)
+// return:
+//        true, if the cartesian point is already defined
+bool HasCartesianPointAnAlias(int number, Vector v, int vertex) {
```

Comment:
```suggestion
bool StepFileWriter::HasCartesianPointAnAlias(int number, Vector v, int vertex) {
```

Ad so on for all newly added functions.